.rubocop.yml

```diff
@@ -4,12 +4,12 @@ Style/ClassAndModuleChildren:
 Style/FrozenStringLiteralComment:
   Enabled: false
 
+Style/SymbolArray:
```

Comment:
I went back and forth on this for a long time but I came down on 'array of symbol syntax is not super helpful'. (Ruby will let you do `%i[thing thing2 thing3]` and it'll be interpreted as `[:thing, :thing2, :thing3]` but I think it's non-intuitive, and if I were coming in cold I think I'd be like 'what on earth does that mean'. So, shutting it off, and moving it back in a couple places.